base/applications/rapps/include/rosui.h

```diff
@@ -302,54 +300,78 @@ class CUiBox
         *newRect = currentRect;
     }
 
-
 public:
+    //virtual ~CUiBox() {} // Needed for current CUiCollection implementation.
+
+    // Override in subclass
     virtual VOID ComputeMinimalSize(SIZE* size)
     {
-        // Override in subclass
         size->cx = max(size->cx, 0);
         size->cy = min(size->cy, 0);
-    };
+    }
 
+    // Override in subclass
     virtual VOID ComputeContentBounds(RECT* rect)
     {
-        // Override in subclass
-    };
+    }
 
-    virtual DWORD_PTR CountSizableChildren()
+    // Override in subclass
+    virtual INT CountSizableChildren()
     {
-        // Override in subclass
         return 0;
-    };
+    }
 
+    // Override in subclass
     virtual HDWP OnParentSize(RECT parentRect, HDWP hDwp)
     {
-        // Override in subclass
-        return NULL;
-    };
+        /* Pass hDwp down the call chain */
+        return hDwp;
```

Comment:
This was a bug, I think.

Reply:
no, it was telling to implement it in a subclass, not to use it as-is.

Reply:
Sure, however it's a default implementation, and suppose that for some reason a UI object doesn't need resizing, then it wouldn't have to override that function, and would just need to pass the resize-handle down the call chain (that would be a sane behaviour). With the old code, it would need to override the function just to do that (even if it doesn't need to implement its resize), otherwise it would break the defer-window handle passing around.